elasticsearch/includes/classes/class-health-job.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Automattic\VIP\Elasticsearch;
+
+use Automattic\VIP\Elasticsearch\Health as Health;
+
+class HealthJob {
+
+	/**
+	 * The name of the scheduled cron event to run the health check
+	 */
+	const CRON_EVENT_NAME = 'vip_search_healthcheck';
+
+	/**
+	 * Custom cron interval name
+	 */
+	const CRON_INTERVAL_NAME = 'vip_half_hour';
```

Comment:
I wonder if we should call this something generic so that it's easier to change the time, if needed?

```suggestion
	const CRON_INTERVAL_NAME = 'vip_search_healthcheck_interval';
```

Reply:
Shouldn't we also change the interval display text too?

Reply:
Yep, good though 👍

Reply:
Changed in 184041da639857bf542743e1bd729c5d043705d2